Bio/SeqRecord.py

```diff
@@ -652,8 +653,14 @@ def __str__(self):
             lines.append(
                 "Per letter annotation for: " + ", ".join(self.letter_annotations)
             )
-        # Don't want to include the entire sequence
-        lines.append(repr(self.seq))
+        try:
+            bytes(self.seq)
+        except UndefinedSequenceError:
+            pass
```

Comment:
Maybe add something like "Undefined sequence of length N" here? See also usage in doctests.

Reply:
done